src/main/java/io/strimzi/kafka/bridge/config/BridgeConfig.java

```diff
@@ -90,6 +123,30 @@ public String getBridgeID() {
         }
     }
 
+    /**
+     * @return the metric system to be used in the bridge
+     */
+    public String getMetrics() {
```

Comment:
Would this better describe what it does?

```suggestion
    public String getMetricsType() {
```

Also, as you defined the Enum, should you use it instead?

Reply:
I agree. Should we do the same with the pre existing `getTracing` method?

Reply:
Now I remember why it was named `getTracing`, because the corresponding property is just  `tracing`. So you used `getMetrics`, because of the corresponding property is named just `metrics`.
Not strong opinion, we could leave the consistency with the corresponding property and having `getMetrics` but of course using the enum. But as I said, not strong opinion on this.